Add backprop support for qutrit operations (#4790)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

* `qml.draw` and `qml.draw_mpl` now render operator ids.
  [(#4749)](https://github.com/PennyLaneAI/pennylane/pull/4749)

> * `default.qutrit` is now differentiable via backprop using TRX, TRY, and TRZ

Reviewer: ```suggestion
* `default.qutrit` is now differentiable via backprop using TRX, TRY, and TRZ
  [(#4790)](https://github.com/PennyLaneAI/pennylane/pull/4790)
```
If you would like, don't forget to add yourself as a contributor at the bottom of this doc!

* `qml.draw` and `qml.draw_mpl` now render operator ids.
  [(#4749)](https://github.com/PennyLaneAI/pennylane/pull/4749)

> * `default.qutrit` is now differentiable via backprop using TRX, TRY, and TRZ

Reviewer 2: ```suggestion
*   `TRX`, `TRY`, and `TRZ` are now differentiable via backprop on `default.qutrit`
```
